doc/yast-storage-requirements.md

```diff
@@ -0,0 +1,348 @@
+# YaST Storage Features and Requirements
+
+This is about *libstorage* and the *yast-storage* Ruby code.
+
+This document was created as a summary of a meeting held in Nuernberg in February 2016
+
+##AIs summary
+
+* AI Lukas: ML: libstorage-ng@opensuse.org
+* AI PM: Clarify requirements on FAT partitions
+* AI lmb, Holgi: Claryfy requirements on filesystems (see below)
+* AI Architecture maintainers: Clarify which configurations of RAID are bootable, possibly under which conditions
+* AI: Veryfy support of resizing logical volumes for LVM
+* AI: Clarify why we do not support encryption of root filesystem, which tools would possibly need to be fixed
+* AI: speak with different parties which requirements they have on automated proposal (Cloud, SES, SAP, ...)
+* AI: Define when dedicated boot partition is needed before merging into the 
+* AI: Find otu if EVMS support for upgrade is necessary (when was it last supported?)
+* AI: Provide specifics for zFCP devices
+* AI: Define requirements on /boot/zipl partition
+* AI: Provide details about specific tools for first sector writing, reasoning for two copies of kernel (why two?) for DASD
```

Comment:
Well, the AI is about finding out and documenting the limitations (actually, the bug is a good starting pointer). Latest when we start the DASD implementation, I would assume this AI to be clarified and documented for the purpose of implementing the right limitations.